front/mapping.form.php

```diff
@@ -29,6 +29,7 @@
  */
 
 require '../../../inc/includes.php';
+Session::checkLoginUser();
 
 /* Update mappings */
 if (isset($_POST["update"])) {
```

Comment:
we need to add this check here

Session::haveRight('plugin_datainjection_model', UPDATE)